plugins/qeta/src/api/types.ts

```diff
@@ -108,3 +108,25 @@ export interface AttachmentResponse {
 }
 
 export type AttachmentResponseBody = AttachmentResponse | ErrorResponse;
+
+export interface StatisticResponse {
+  ranking: Statistic[];
+  loggedUser?: Statistic;
+}
+
+export interface Statistic {
```

Comment:
These also need some refactoring and could be placed in qeta-common to prevent duplicating these in backend and frontend